### PR TITLE
New count_occurrences

### PR DESCRIPTION
**Pull request by KostiantinPastukh**

### CODE REVIEW

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
The code logic is legit, however, PR contains redundant changes. Removal was asked.

File: app/main.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+    #
+    # count_occurrences("letter", "t") == 2
+    # count_occurrences("abc", "a") == 1
+    # count_occurrences("abc", "d") == 0
+    # count_occurrences("ABC", "a") == 1
+    #
+    # :param phrase: phrase to count in it
+    # :param letter: letter to find occurrences of it
+    # :return: count occurrences of letter in phrase
+    # """
```
[the remark this answers is not in the document]
Comment (KostiantinPastukh) [reply app/main.py]: Like that?

File: app/main.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
```
Comment (pavlejviki) [inline app/main.py:16]: Still not fixed. Just uncomment the docstrings.

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by pavlejviki — APPROVED** [review]
Well done!

**Pull request closed without merging**